GVRf/Framework/framework/src/main/java/org/gearvrf/IAssetEvents.java

```diff
@@ -0,0 +1,70 @@
+/* Copyright 2016 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gearvrf;
+
+/**
+ * Interface for processing asset load events.
+ * The asset loader raises an event whenever a texture
+ * or a model is loaded. It also raises an event when
+ * the entire asset (models and textures) is loaded.
+ * This interface lets you observe  these events.
+ */
+public interface IAssetEvents extends IEvents
+{
+    /**
+     * Called when a model is successfully loaded.
```

Comment:
I changed the JavaDoc to explain onModelLoaded does not wait for textures while onAssetLoaded does.

From: rahul [mailto:notifications@github.com]
Sent: Thursday, June 2, 2016 5:55 PM
To: Samsung/GearVRf <GearVRf@noreply.github.com>
Cc: Nola Donato <nola.donato@samsung.com>; Author <author@noreply.github.com>
Subject: Re: [Samsung/GearVRf] Wait for textures to be loaded before adding model to the scene (#597)


In GVRf/Framework/framework/src/main/java/org/gearvrf/IAssetEvents.java<https://github.com/Samsung/GearVRf/pull/597#discussion_r65643443>:

> + * See the License for the specific language governing permissions and

> + * limitations under the License.

> + */

> +package org.gearvrf;

> +

> +/**

> + * Interface for processing asset load events.

> + * The asset loader raises an event whenever a texture

> + * or a model is loaded. It also raises an event when

> + * the entire asset (models and textures) is loaded.

> + * This interface lets you observe  these events.

> + */

> +public interface IAssetEvents extends IEvents

> +{

> +    /**

> +     * Called when a model is successfully loaded.

The javadocs are identical for onAssetLoaded and onModelLoaded. Makes it harder to understand what is called when.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/Samsung/GearVRf/pull/597/files/fb58b0721681e130ce48eec2b79e1fc44b7ecf78#r65643443>, or mute the thread<https://github.com/notifications/unsubscribe/AQw1S-ZDHvhSaI08NKzpZDo2zW2DqhAVks5qH3tmgaJpZM4ItA3y>.